docs/source/transforms.rst

```diff
@@ -76,6 +76,11 @@ Transforms on PIL Image and torch.\*Tensor
 .. autoclass:: Pad
     :members:
 
+.. include:: gen_modules/backreferences/torchvision.transforms.Pad.examples
```

Comment:
there's a better way to do this with more recent versions of sphinx-gallery (and possibly even with this one, I didn't dive too much into it)